src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -140,7 +142,7 @@
 				variant="secondary"
 				v-if="isInBasket"
 				v-kv-track-event="['Lending', 'click-Read more', 'checkout-now-button-click', loanId, loanId]"
-				to="/basket"
+				:to="customCheckoutRoute.length > 0 ? customCheckoutRoute : '/basket'"
```

Comment:
You can just check for falsy on the empty string here, you don't need to check for the length.